dvc/command/experiments.py

```diff
@@ -150,7 +150,12 @@ def _collect_rows(
 
     new_checkpoint = True
     for i, (rev, exp) in enumerate(experiments.items()):
-        queued = str(exp.get("queued") or "")
+        if exp.get("running"):
+            state = "Run"
+        elif exp.get("queued"):
+            state = "Queue"
```

Comment:
Should it be `Running`/`Queued`? Just feels a bit odd. Maybe this has been discussed somewhere. Just wondering.

Reply:
Running/Queued makes more sense but I'm not really sure what's best here given that table width is a concern